staging/src/k8s.io/kubectl/pkg/cmd/debug/debug_test.go

```diff
@@ -346,7 +346,7 @@ func TestGenerateDebugContainer(t *testing.T) {
 				tc.pod = &corev1.Pod{}
 			}
 
-			applier, err := NewProfileApplier(tc.opts.Profile)
+			applier, err := NewProfileApplier(tc.opts.Profile, &keepFields{})
```

Comment:
I think, it would be better to use builder pattern. i.e.

```go
applier, err := NewProfileApplier(tc.opts.Profile)
if o.KeepAnnotations{
applier.WithKeepAnnotations()
}
...
```

Reply:
Thank you for your comment.
I've tried to fix.